src/components/layout/GenericLayout/Footer/config.ts

```diff
@@ -1,13 +1,13 @@
 export const footerConfig = {
   isBeta: false,
   url: {
-    web: 'https://github.com/gnosis/gp-ui/tree/v',
+    web: 'https://github.com/cowprotocol/explorer/tree/v',
```

Comment:
This links doesn't work:
https://github.com/cowprotocol/explorer/tree/v2.7.0

I assume that it will work when we approve the release 2.7.0

Reply:
You're right, it's already working :runner: